### PR TITLE
Revcontent, support render-start (#5234)

### PR DESCRIPTION
**Pull request by e2corporation**
Add configuration support for accessing **renderStart()** API. See #5234 for additional details.

### CODE REVIEW

**Comment by e2corporation** [comment]
@dvoytenko @lannka Will this PR make the canary release? Thank you.

**Comment by lannka** [comment]
@e2corporation if we can get it in tmr

**Comment by e2corporation** [comment]
@lannka Any luck adding this one to the release queue? Thanks.

File: examples/ads.amp.html [hunk 1]
```diff
@@ -942,15 +942,26 @@ <h2 id="purch">Purch 300x250</h2>
...
+        (max-width: 736px) 1151px,
+        (max-width: 768px) 633px,
+        (max-width: 1024px) 711px,
+        86vw"
       data-wrapper="rcjsload_2ff711"
-      data-endpoint="trends.revcontent.com"
       data-id="203">
-    <div placeholder></div>
-    <div fallback></div>
+    <div placeholder="">Loading ...</div>
```
Comment (lannka) [inline examples/ads.amp.html]: please keep the old placeholder and fallback

**Comment by e2corporation** [comment]
Updated to use old tags.

**Comment by lannka** [comment]
screen shot 2016-11-30 at 1 51 04 pm

@e2corporation I got no ad but this error when playing with the example ad. Is it expected due to targeting criteria? Are we able to make it working locally?

**Comment by e2corporation** [comment]
You would need to specify an alternate endpoint on the amp tag,
data-endpoint="trends-s1.revcontent.com" to use one of our test servers.

Production endpoint is used by default and has rules on where it will allow
a fill to be delivered, your network appears to be blocked.

On Nov 30, 2016 4:53 PM, "Hongfei Ding" <notifications@github.com> wrote:

[image: screen shot 2016-11-30 at 1 51 04 pm]
<https://cloud.githubusercontent.com/assets/8496897/20772860/1da79b10-b704-11e6-9b74-81ec2b413046.png>

@e2corporation <https://github.com/e2corporation> I got no ad but this
error when playing with the example ad. Is it expected due to targeting
criteria? Are we able to make it working locally?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/ampproject/amphtml/pull/6332#issuecomment-264007441>,
or mute the thread

**Comment by lannka** [comment]
I see, @e2corporation can you make it to the example?

**Comment by e2corporation** [comment]
Typically our stage servers are for internal use and not to be used on our
example panel.

Are you testing our ad on a cloud build of amp or a local one?

**Comment by lannka** [comment]
On my local. When you say our network is blocked, do you mean our IPs are banned?

**Comment by e2corporation** [comment]
If you check the xhr tab filter for "revcontent.com" to see the message
returned.

**Comment by lannka** [comment]
@e2corporation let me know if you want the PR to be merged or you have more things to add

**Comment by e2corporation** [comment]
@lannka Let me add another optional parm to data validation.

**Comment by e2corporation** [comment]
@lannka Will this PR make the release? Build error from Travis CI does not appear related to the code changes. Thanks.

**Comment by e2corporation** [comment]
@lannka @dvoytenko Any chance we can get this into master today? Thank you.

**Pull request merged**